pulumi/grapl/__main__.py

```diff
@@ -338,6 +340,13 @@ def main() -> None:
             nomad_agent_security_group_id=nomad_agent_security_group_id,
         )
 
+        postgres = Postgres(
+            name="postgres",
```

Comment:
Are we only going to have one Postgres instance for all of Grapl, or one per service? If the latter, we should name this `plugin-registry`.

Reply:
Open question. I asked Colin. I'm going to submit the rest of the updated PR without addressing this [yet]